tests/unittests/api/helpers/test_order.py

```diff
@@ -19,7 +19,7 @@ def test_should_expire_outdated_order(self):
         with app.test_request_context():
             obj = OrderFactory()
             obj.created_at = datetime.now(timezone.utc) - timedelta(
-                minutes=ticketing.TicketingManager.get_order_expiry() + 10)
+                minutes=obj.event.order_expiry_time)
```

Comment:
First create an event using the factory for event. Then set it on the order obj.